jetty-osgi/test-jetty-osgi-webapp-resources/pom.xml

```diff
@@ -70,14 +60,6 @@
           </archive>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-deploy-plugin</artifactId>
-        <configuration>
-          <!-- must not deploy -->
```

Comment:
I have to say I was confused here and not sure to believe the comment or the code. :)